.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
```

Comment:
unsure about why we need this because we are not doing a lot of js

Reply:
But we are using JS though.